theano/tensor/extra_ops.py

```diff
@@ -11,6 +11,43 @@
 tensor = basic
 
 
+class cpu_contiguous(theano.Op):
+    def __eq__(self, other):
+        return type(self) == type(other)
+
+    def __hash__(self, other):
+        return hash(type(self))
+    
+    def make_node(self, x):
+        x_ = theano.tensor.as_tensor_variable(x)
+        return theano.Apply(self, [x_], [x_.type()])
+    
+    def perform(self, node, inputs, output_storage):
+        x, = inputs
+        y = output_storage[0][0]
```

Comment:
This won't work.

You would have to say:
```python
y = output_storage[0]
# compute result
y[0] = x
```
Otherwise you will just overwrite the value of the local variable `y` and not change the storage list.